x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/Order.java

```diff
@@ -18,7 +19,19 @@
 public class Order extends Expression {
 
     public enum OrderDirection {
-        ASC, DESC
+        ASC, DESC;
+
+        public static OrderDirection fromString(String direction) {
```

Comment:
The `direction == null || direction.isEmpty()` is trappy - this is an EQL default that leaks into QL, which should not be the case.
Also this utility method of converting String to enum has questionable value...

Reply:
I agree, it can be confusing to have this in `ql` module since the functionality is EQL specific.